mmcv/cnn/bricks/transformer.py

```diff
@@ -36,35 +38,74 @@ def build_transformer_layer_sequence(cfg, default_args=None):
     return build_from_cfg(cfg, TRANSFORMER_LAYER_SEQUENCE, default_args)
 
 
+def bnc_to_nbc(forward):
+    """This function can adjust the shape of dataflow('key', 'query', 'value')
+    from batch_first (batch, num_query, embed_dims) to num_query_first
+    (num_query ,batch, embed_dims)"""
+
```

Comment:
We may also add example usage for this function.

Reply:
Sure

Reply:
I move it to `MultiheadAttention` as a private function, because it only can be used in our `MultiheadAttention`.